index.php

```diff
@@ -75,10 +75,13 @@
 require_once($sourcedir . '/Subs-Auth.php');
 require_once($sourcedir . '/Errors.php');
 require_once($sourcedir . '/Load.php');
+require_once($sourcedir . '/Security.php');
```

Comment:
Now that this is here, please remove the duplicate line of code that exists further down in the file.